charts/argo-cd/templates/argocd-configs/argocd-secret.yaml

```diff
@@ -14,7 +14,11 @@ metadata:
 type: Opaque
 {{- if or .Values.configs.secret.githubSecret (or .Values.configs.secret.gitlabSecret .Values.configs.secret.bitbucketUUID .Values.configs.secret.bitbucketServerSecret .Values.configs.secret.gogsSecret .Values.configs.secret.argocdServerAdminPassword .Values.configs.secret.argocdServerTlsConfig .Values.configs.secret.extra) }}
 # Setting a blank data again will wipe admin password/key/cert
+{{- if .Values.configs.secret.encrypt }}
 data:
+{{- else }}
+stringData:
```

Comment:
This will not work, look at some lines below. We use `| b64enc` everywhere.

Reply:
Maybe use something like
`admin.password: {{- if .Values.configs.secret.encrypt }}{{ .Values.configs.secret.argocdServerAdminPassword | b64enc }} {{else}} {{ .Values.configs.secret.argocdServerAdminPassword }}`